### PR TITLE
Issue #3375: Hotfix for ImageToolkitGdTestCase failing on newer PHP 7 and 7.1 releases

### PR DESCRIPTION
**Pull request by klonos**
https://www.drupal.org/project/drupal/issues/2918570#comment-12365552

### CODE REVIEW

File: core/modules/simpletest/tests/image.test [hunk 1]
```diff
@@ -407,7 +409,8 @@ class ImageToolkitGdTestCase extends BackdropWebTestCase {
         // dimensions. For the test images, the dimensions will be 1 pixel
         // smaller in both dimensions (though other tests have shown a
         // difference of 0 to 3 pixels in both dimensions.
-        // @todo: Add an upper limit version check if this is fixed in PHP.
+        // @todo: The PHP bug was fixed in PHP 7.0.26 and 7.1.12. Change the code
```
Comment (jenlampton) [inline core/modules/simpletest/tests/image.test:412]: We can remove this @todo, it's for Drupal 8.

Comment (jenlampton) [reply core/modules/simpletest/tests/image.test:412]: Oh, actually it looks like cleanup for our tests. It can stay :)

**Comment by jenlampton** [comment]
RTBC :)

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x and 1.11.x. It's a little strange this doesn't actually fix anything, it just adds a documentation link and suppresses displaying an option, but it's good for consistency so we may be able to apply later patches.

**Pull request merged**